### DISCUSSION TITLE
how to insert many data at first runtime? (use already created sql query)

### DISCUSSION THREAD

**Discussion by jerryp123123** [simolus3/drift#2300] [Q&A]
Hello!

How to insert many data first runtime ?
I want use to already created SQL Query

Is there a way?
almost 80000 row data...

**Comment by simolus3** [comment]
Sure, you can use [`customInsert`](https://drift.simonbinder.eu/api/drift/databaseconnectionuser/custominsert) to run any SQL statement with drift.

If you want to run the same statement many times with different parameters to insert the 80,000 rows, you can use a [`customStatement`](https://drift.simonbinder.eu/api/drift/batch/customstatement) in a [batch](https://drift.simonbinder.eu/api/drift/databaseconnectionuser/batch).
